io/include/pcl/compression/impl/entropy_range_coder.hpp

```diff
@@ -265,8 +263,8 @@ pcl::StaticRangeCoder::encodeIntVectorToStream (std::vector<unsigned int>& input
       }
 
       // init new frequency range with zero
-      memset (&cFreqTable_[static_cast<std::size_t> (oldfrequencyTableSize + 1)], 0,
-              sizeof(std::uint64_t) * static_cast<std::size_t> (frequencyTableSize - oldfrequencyTableSize));
+      std::fill_n(&cFreqTable_[static_cast<std::size_t> (oldfrequencyTableSize + 1)],
```

Comment:
Is this static cast needed?

Reply:
All kinds of funky casting going on in this file; I removed this one.